app/index.html

```diff
@@ -10,9 +10,9 @@
 
     <link rel="stylesheet" href="css/main.css">
   </head>
-  <body ng-app='theshelf'>
-    <side-content></side-content>
-    <div ui-view></div>
+  <body class='Site' ng-app='theshelf'>
```

Comment:
Well there isn't any indication that there will be any.
I felt like simplifying.
What would you call it anyway? It's the site's layout.
And besides, it's used in one place alone (here), if the need comes to change it, then it couldn't be easier.